django/contrib/admin/apps.py

```diff
@@ -11,6 +13,9 @@ class SimpleAdminConfig(AppConfig):
     verbose_name = _("Administration")
 
     def ready(self):
+        if settings.DEBUG:
```

Comment:
I think there's little need to keep this -- I don't think any other checks are controlled by DEBUG.